plugins/ipc-rules/ipc-utility-methods.hpp

```diff
@@ -281,6 +283,16 @@ class ipc_rules_utility_methods_t
             return wf::ipc::json_error("Options must be an object!");
         }
 
+        const char *config_file     = std::getenv("WAYFIRE_CONFIG_FILE");
```

Comment:
I think strictly speaking we don't really need to print the config file every time, we say this at the beginning of the log. So we could just say that we won't laod from the config file. Consider that for example config backends could also use a different config file altogether not the ini file.